packages/element/package.json

```diff
@@ -0,0 +1,30 @@
+{
+	"name": "@wordpress/element",
+	"version": "0.0.1",
+	"description": "Element module for WordPress",
```

Comment:
Wondering if `React` should be included in the title for < cough >~~SEO~~</ cough > marketing/discoverability purposes?

> `Element React module for WordPress`